radio/src/targets/taranis/board.h

```diff
@@ -650,7 +650,7 @@ extern uint16_t adcValues[NUM_ANALOGS];
   #define BATTERY_WARN                  66 // 6.6V
   #define BATTERY_MIN                   67 // 6.7V
   #define BATTERY_MAX                   83 // 8.3V
-#elif defined(RADIO_T8) || defined(RADIO_TLITE)
+#elif defined(RADIO_T8) || defined(RADIO_TLITE) 
```

Comment:
Could you please remove the space at the and of the line?

Reply:
OK,This is my negligence. I will delete these spaces.